packages/lib-webpack/package.json

```diff
@@ -3,48 +3,27 @@
   "version": "1.0.0-alpha2",
   "description": "Allows building plugin assets with webpack",
   "license": "Apache-2.0",
-  "type": "module",
   "repository": {
     "type": "git",
     "url": "https://github.com/openshift/dynamic-plugin-sdk.git",
     "directory": "packages/lib-webpack"
   },
-  "files": [
-    "dist/index.js",
-    "dist/index.d.ts"
-  ],
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
   "scripts": {
     "prepack": "yarn build",
     "prepublishOnly": "yarn test",
-    "build": "rm -rf dist && rollup -c",
+    "build": "rm -rf dist && yarn run -T rollup -c",
     "lint": "echo 'Run the linter!'",
     "test": "echo 'Write some tests!'"
   },
   "peerDependencies": {
-    "@openshift/dynamic-plugin-sdk": "^1.0.0",
     "webpack": "^5.69.1"
   },
   "dependencies": {
     "glob": "^7.2.0",
+    "lodash": "^4.17.21",
     "lodash-es": "^4.17.21",
```

Comment:
We should pick one and remove the other. Are we using `lodash-es` in this package?

Reply:
This should be fixed, `lib-webpack` is built against `lodash-es` but the CommonJS `require()` statement is modified to import `lodash`.

`lodash-es` is ES modules friendly (and therefore modern, tree shaking and optimization friendly) build of the standard CommonJS `lodash` build. Their APIs are identical.